apps/coscrad-frontend/src/components/resources/playlists/playlist-index.presenter.tsx

```diff
@@ -38,10 +39,12 @@ export const PlaylistIndexPresenter = ({ entities: playlists }: PlaylistIndexSta
         { propertyKey: 'id', headingLabel: 'Link' },
         { propertyKey: 'episodes', headingLabel: 'Number of Episodes' },
         { propertyKey: 'name', headingLabel: 'Playlist' },
+        { propertyKey: 'url', headingLabel: 'Play' },
```

Comment:
One more time- there is **no** `url` property on `PlaylistViewModel`. There is an `episodes` property of type `PlaylistEpisodes[]`- and each of these has a `mediaItemUrl`.

Reply:
this has been changed